src/dhtproto/client/legacy/internal/helper/RetryHandshake.d

```diff
@@ -24,74 +24,48 @@ class RetryHandshake
 
     import ocean.io.select.EpollSelectDispatcher;
     import ocean.io.select.client.TimerEvent;
+    import ocean.util.log.Logger;
 
-    /***************************************************************************
-
-        Timer to retry the handshake
-
-    ***************************************************************************/
+    /// Logger for instances of this class.
+    /// (Note that this assumes there is only one instance of this class. This
+    /// is the case in practice. If we ever TODO TODO)
```

Comment:
`TODO TODO`?!

Reply:
Oops. That was obviously meant to be a message from my Friday self to my self now. I completely missed it :D

Reply:
Updated.